cmd/txottl/txottlparse.go

```diff
@@ -230,23 +285,60 @@ func ReadTTLdb(sig chan bool) error {
 			fmt.Printf("done with height %d\n", height)
 
 			height++
+			//Only write where we left off after a block has been finished
+			ttlfilesync.WriteAt(U32tB(uint32(position))[:], 0)
 
 			// start next block
 			// wipe all block txs
 			blocktxs = []*txotx{new(txotx)}
 
+			//Check if stopSig is no longer false
+			//stop = true makes the loop exit
+			if stopSig == true {
```

Comment:
same idea here; replacing this with
```
	        select {
		case stop = <-stopGoing:
		default:
		}
```
lets you get rid of the goroutine on line 188.  It also would let you get rid of the whole `stopTxottl` routine if it were 

```
	        select {
		case stop = <-sig:
		default:
		}
```

which seems simpler as then there's 1 channel instead of 3

Reply:
Wait, not 100% sure on that last one.  In `stopTxottl` the bool coming out of `sig` is just dropped, so the thing I wrote would only work when `sig` has a true in it, but not a false.  

So could still work, just that's a difference to be aware of.

Reply:
Implemented requested changes for both #28 and #29